### DISCUSSION TITLE
How can I fix "Vue packages version mismatch" error in Vue3

### DISCUSSION THREAD

**Discussion by Leopard-CC** [community/community#36447] [Other Feature Feedback, Questions, & Ideas] [answered]
When I run npm run dev on my nuxt js, I get the following error:When I run npm run dev on my nuxt js, I get the following error:

```
FATAL                                                                                                                                                          14:16:02  

Vue packages version mismatch:

- vue@3.2.40
- vue-server-renderer@2.7.12

This may cause things to work incorrectly. Make sure to use the same version for both.



  Vue packages version mismatch:

  - vue@3.2.40
  - vue-server-renderer@2.7.12

  This may cause things to work incorrectly. Make sure to use the same version for both.

  at Object.<anonymous> (node_modules\vue-server-renderer\index.js:8:9)
  at Module._compile (node:internal/modules/cjs/loader:1126:14)
  at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
  at Module.load (node:internal/modules/cjs/loader:1004:32)
  at Function.Module._load (node:internal/modules/cjs/loader:839:12)
  at Module.require (node:internal/modules/cjs/loader:1028:19)
  at require (node:internal/modules/cjs/helpers:102:18)
  at Object.<anonymous> (node_modules\@nuxt\vue-renderer\dist\vue-renderer.js:20:27)
  at Module._compile (node:internal/modules/cjs/loader:1126:14)
  at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
```

Anyone could help me?

**Comment by softdream815** [comment, marked as answer]
Nuxt2 and Vue3 are not compatible together, one will use a package version which is incompatible with the other.

And anyway, Nuxt2 is only for a Vue2 usage. If you want to use Vue3, you need to use Nuxt3.

But if you use Nuxt (2 or 3), you don't need to install Vue because it's already baked-in into Nuxt.

It's important that you check that the packages you're using are compatible with their version.

Reinstalling the packages will not fix anything for the simple reason that your package.json is still wrong (some package versions are not compatible with others) waiting for a fix: choosing versions that work all together.

Also, you probably can even remove vue-server-renderer IMO since it is probably a dependency of another package.

PS: Github co-pilot is not related. Dotenv doesn't need to be installed because it's already baked-in into Nuxt. And you should probably not install Vuetify and the module of the same name at the same time. You also have eslint-config-prettier while not using ESlint at all apparently.

You can use this answer to configure ESlint + Prettier properly: https://stackoverflow.com/a/68880413/8816585

**Reply by Leopard-CC** [reply]
Thank you for quick answer
It works fine :)
